Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/EditorComponentModeBus.h

```diff
@@ -142,6 +142,9 @@ namespace AzToolsFramework
             /// has the same Component on it, move all Components into ComponentMode.
             virtual void AddSelectedComponentModesOfType(const AZ::Uuid& componentType) = 0;
 
+            // Switches to the ComponentMode of input component type immediately.
```

Comment:
uber nit: Sorry this need three `/` to be a Doxygen comment (this is one of the reasons why we now prefer `//!` but we don't need to make that change in this file yet (could be a separate formatting style change later)